src/Umbraco.Web.UI.Client/src/common/services/editor.service.js

```diff
@@ -173,13 +173,14 @@ When building a custom infinite editor view you can use the same components as a
 
 
         // events for backdrop
-        eventsService.on("appState.backdrop", function (name, args) {
+        var backgroundEvent = eventsService.on("appState.backdrop", function (name, args) {
             if (args.show === true) {
                 blur();
             } else {
                 focus();
             }
         });
+       
```

Comment:
no access to $scope, need another approach for destruction

Reply:
@nzdev service is a singleton, so no need to worry about tear down